src/components/menu/Menu.jsx

```diff
@@ -19,25 +30,32 @@ const Menu = () => {
           </div>
         )}
 
+        {isOpen && <div className="overlay" onClick={closeMenu} />}
+        
         {isOpen && (
-          <>
-            <div className="overlay" onClick={closeMenu} />
-            <div className="side-menu">
-              <button className="close-button" onClick={closeMenu}>
-                &times; 
-              </button>
-              <h1 className='name'>Levantini</h1>
-              <ul className="menu-options">
-                <li><MdHome /> Home</li>
-                <li><LiaCrownSolid /> Leaderboard</li>
-                <li><IoIosLogOut /> Log out</li>
-              </ul>
-            </div>
-          </>
+          <div className="side-menu">
+            <button className="close-button" onClick={closeMenu}>
+              &times; 
+            </button>
+            <h1 className='name'>Levantini</h1>
+            <ul className="menu-options">
+              <li onClick={closeMenu} style={{ cursor: 'pointer' }}>
+                <Link to="/" style={{ textDecoration: 'none', color: 'inherit' }}>
```

Comment:
take the url link from our constants src/routes/routeConstants.js

Reply:
did't understand